snapshots/papers/2023-05-26/ray_et_al_2019.xlsx.dvc

```diff
@@ -0,0 +1,31 @@
+meta:
+  namespace: papers
+  short_name: ray_et_al_2019
+  name: Climate change has likely already affected global food production - Ray et
+    al. (2019)
+  version: '2023-05-26'
+  publication_year: 2019
+
+  publication_date: '2019-05-31'
+  source_name: Ray et al. (2019)
+  source_published_by: 'Ray DK, West PC, Clark M, Gerber JS, Prishchepov AV, et al.
+    (2019) Climate change has likely already affected global food production. PLOS
+    ONE 14(5): e0217148. https://doi.org/10.1371/journal.pone.0217148'
+  url: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0217148
+  source_data_url: https://doi.org/10.1371/journal.pone.0217148.s017
+  file_extension: xlsx
+  license_url: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0217148
+  license_name: Creative Commons Attribution License
+  date_accessed: 2023-05-26
+  is_public: true
+  description: |
+    Ray et al. (2019) modelled the impact of observed climate change on yields of the top ten global crops. These ten crops account for ~83% of global kilocalorie production.
```

Comment:
I don't think it's that clear what period the paper and the data values are covering? Is the start 1974 and the values shown here are the average of 2009-13? I think it would be useful to clarify what exactly is shown.

Reply:
I agree, it's not clear. I added a note in the code, but for now I'll simply keep the dataset description that we have [in grapher](https://owid.cloud/admin/datasets/5344).